README.md

```diff
@@ -130,6 +130,35 @@ export default Route.extend({
 });
 ```
 
+### Launch Darkly Service
+
+ember-launch-darkly automatically injects the launch darkly service in to all
+`route`s, `controller`s and `component`s as `ldService`. This means
+that it can be accessed without an explicit injection, like so:
+
+```js
+// /app/application/route.js
+
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model() {
+    let user = {
+      key: 'aa0ceb',
+      anonymous: true
+    };
+
+    return this.ldService.initialize(user);
```

Comment:
Curious why `ldService` and not `launchDarkly`?

Reply:
Original rationale - Initially, it was because the babel transform was adding the injection to objects that were using the `variation` helper. In order to sidestep conflicts whereby the user have explicitly injected the `launchDarkly` service themselves, it felt like another name that was specific to this transform was more useful. `ldService` obviously isn't unique but the thinking was that the general pattern, for a well named service like `launchDarkly`, is simply to use `launchDarkly: service()`.

So, that was the history. However, now that we are auto injecting the service from an initializer, this is less of an issue. I was only thinking through this the other night. I could just auto inject `launchDarkly` as is.

In that same thought, I had also, I did start wondering whether not being specific about launch darkly might be worth thinking about. ie, using `featureFlags` or `features`. Could be something to consider too. [kategengler/ember-feature-flags](https://github.com/kategengler/ember-feature-flags) uses `features`.

Having said that, the API of the service, `launchDarkly.variation()` is LD specific anyway. If only there was a standard way to talk about feature flags eh?

So, yeh, I'll switch it to `launchDarkly` now as I think that makes sense.

Reply:
Have renamed